test/libsolidity/semanticTests/operators/custom/all_operators.sol

```diff
@@ -0,0 +1,105 @@
+type Int is int128;
+using {
+    bitor as |, bitand as &, bitxor as ^, bitnot as ~,
+    add as +, sub as -, unsub as -, mul as *, div as /, mod as %,
+    eq as ==, noteq as !=, lt as <, gt as >, leq as <=, geq as >=
+} for Int;
+
+function uw(Int x) pure returns (int128) {
+    return Int.unwrap(x);
+}
+function w(int128 x) pure returns (Int) {
+    return Int.wrap(x);
+}
+function bitor(Int, Int) pure returns (Int) {
+    return w(10);
+}
+function bitand(Int, Int) pure returns (Int) {
+    return w(11);
+}
+function bitxor(Int, Int) pure returns (Int) {
+    return w(12);
+}
+function bitnot(Int) pure returns (Int) {
+    return w(13);
+}
+function add(Int x, Int) pure returns (Int) {
+    return w(uw(x) + 10);
+}
+function sub(Int, Int) pure returns (Int) {
+    return w(15);
+}
+function unsub(Int) pure returns (Int) {
+    return w(16);
+}
+function mul(Int, Int) pure returns (Int) {
+    return w(17);
+}
+function div(Int, Int) pure returns (Int) {
+    return w(18);
+}
+function mod(Int, Int) pure returns (Int) {
+    return w(19);
+}
+function eq(Int x, Int) pure returns (bool) {
+    return uw(x) == 1;
+}
+function noteq(Int x, Int) pure returns (bool) {
+    return uw(x) == 2;
+}
+function lt(Int x, Int) pure returns (bool) {
+    return uw(x) < 10;
+}
+function gt(Int x, Int) pure returns (bool) {
+    return uw(x) > 10;
+}
+function leq(Int x, Int) pure returns (bool) {
+    return uw(x) <= 10;
+}
+function geq(Int x, Int) pure returns (bool) {
+    return uw(x) >= 10;
+}
+
+contract C {
+    function test_bitor() public pure returns (Int) { return w(1) | w(2); }
+    function test_bitand() public pure returns (Int) { return w(1) | w(2); }
```

Comment:
Looks like a typo here: `|` is used instead of `&`.

Reply:
Nice catch!